### PR TITLE
fix pagination issue in sorted listview page

### PR DESCRIPTION
**Pull request by sumandari**
@dimasciput @timlinux 

This PR refers to #103. It does:
- Fix pagination in sorted listview
- Update form_validation when the user uploads a QGIS style XML file that its style-name already exists.
- Update title on Style Type list view page from **All Styles** to its style name

upload_style

### CODE REVIEW

File: qgis-app/styles/templates/styles/style_list.html [hunk 1]
```diff
@@ -1,40 +1,57 @@
...
+        <p>{{ count }} record{{ style_list.count|pluralize:"s" }} found.</p>
     <table class="table table-striped plugins">
         <thead>
             <tr>
                 <th></th>
                 <th>
                     Name
-                    <a href="{{ request.get_path }}?order_by=name"><i class="icon-arrow-up"></i></a>
-                    <a href="{{ request.get_path }}?order_by=-name"><i class="icon-arrow-down"></i></a>
+                    <a href="{{ request.get_path }}?order_by=name" {% if order_by == "name" %}class="sorted"{% endif %}><i class="icon-arrow-up"></i></a>
```
Comment (dimasciput) [inline qgis-app/styles/templates/styles/style_list.html:33]: @sumandari I think you can use template tag for this, https://docs.djangoproject.com/en/3.1/howto/custom-template-tags/. Then we can reuse for all order link.

Comment (sumandari) [reply qgis-app/styles/templates/styles/style_list.html:33]: Thanks  a lot for your advice @dimasciput. I will learn it and replace my primitive approach 😅 on the template  with this one.

**Review by dimasciput — APPROVED** [review]
LGTM

**Comment by timlinux** [comment]
LGTM

**Pull request merged**